README.md

```diff
@@ -12,32 +31,59 @@ A nice Minecraft (non-hacked) client for 1.8.9 (and possibly newer versions in f
 - 1.7 animations
 - Item physics
 - Hypixel additions
-- OptiFine (optional, from optifine.net)
-- Crosshair
-- Quick play (command palette style) (uses games from robere2's mod)
-- Better item tooltips, showing the full description
+- OptiFine (optional, from [its website](https://optifine.net/downloads))
+- Custom crosshairs
+- Quick Play (uses games from robere2's mod)
+- Better item tooltips (like the ones in Bedrock Edition)
 - Chat symbol picker
 - Chat channel picker and status
-- Popup events (based on [this](https://github.com/Sk1erLLC/PopupEvents))
+- Popup events (based on Sk1er's [PopupEvents](https://github.com/Sk1erLLC/PopupEvents))
 - Speedometer
-- Chunk animator (based on [this](https://github.com/lumien231/Chunk-Animator))
+- Animated chunks (based on Lumien's [Chunk Animator](https://github.com/lumien231/Chunk-Animator))
 - Timers
-- Pack folders
-- Replay mod (using [this](https://github.com/ReplayMod/ReplayMod))
-- Entity culling (using an outdated version of [this](https://github.com/tr7zw/EntityCulling)!! the owner adopted a more restrictive license since which is actually fairly understandable)
+- Resource pack folders
+- Replay Mod (using the actual [Replay Mod](https://github.com/ReplayMod/ReplayMod))
+- Entity Culling (using an **outdated version** of tr7zw's [Enity Culling](https://github.com/tr7zw/EntityCulling) mod, since the owner adopted a more restrictive license)
```

Comment:
I kind of liked the sarcastic enthusiasm with "!!" in the old readme, and I don't think bold text is necessary.

Reply:
Ok then